modules/apps/headless/headless-delivery/headless-delivery-api/bnd.bnd

```diff
@@ -3,7 +3,9 @@ Bundle-SymbolicName: com.liferay.headless.delivery.api
 Bundle-Version: 29.0.3
 Export-Package:\
 	com.liferay.headless.delivery.dto.v1_0,\
+	com.liferay.headless.delivery.dynamic.data.mapping,\
 	com.liferay.headless.delivery.resource.v1_0,\
 	com.liferay.headless.delivery.search.aggregation,\
 	com.liferay.headless.delivery.search.filter,\
-	com.liferay.headless.delivery.search.sort
\ No newline at end of file
+	com.liferay.headless.delivery.search.sort,\
+	com.liferay.headless.delivery.util
```

Comment:
Don't know if it is a little bit generic. We can try this way for now. Other option would be `com.liferay.headless.delivery.dto.v1_0.util` (It is possible that if we change the dtos in a v2 of the apis all this classes become useless)